ops/fhir_validations/validations_ignorelist.yml

```diff
@@ -49,3 +49,7 @@ ignore_list:
       error_patterns:
         - ".*ExplanationOfBenefit\\.diagnosis\\.diagnosis\\[x\\].* minimum required.*"  # Missing field (6 errors)
         - ".*ReferralRequest\\.intent.* minimum required.*"                             # Missing field (8 errors)
+    # V2 specific validation issues
+    - file_pattern: .*/v2/.*\.json
+      error_patterns:
+        - ".*ReferralRequest\\.intent.* minimum required.*"
```

Comment:
Is this for sure something to ignore and not something to fix?

Reply:
Yes according to Bill Referral request shouldnt even show up in v4